amf-client/shared/src/test/resources/upanddown/externals.json

```diff
@@ -50,11 +50,16 @@
             "name": "generated",
             "schema": {
               "type": "object",
-              "properties": {
-                "a": {
-                  "type": "string"
+              "x-amf-merge": [
```

Comment:
Looks like original now

Reply:
AMF is the only one that knows how to read a "x-amf-" extension. We are changing the meaning of the shape for any other processor, right?

Reply:
Yes, the thing is that this is a RAML to OAS cycle, a deprecated (?) use case. It is reflecting the AST change. If I resolve the API with CompatibilityPipeline then the output is the same as it was before this change.

Reply:
Great! if it happens only when converting in a wrong way, then should not be a problem.